roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -76,10 +76,9 @@
 
 - name: Apply ceph rgw keystone config
   ansible.builtin.command: |
-    {{ cifmw_cephadm_ceph_cli }} config set global {{ item.key }} {{ item.value }}
+    {{ cifmw_cephadm_ceph_cli }} config assimilate-conf -i {{ cifmw_cephadm_assimilate_conf_container }}
```

Comment:
nit: this variable `cifmw_cephadm_assimilate_conf_container` is confusing to me: I understand this path is something resolved within the cephadm container, but can we rename to something better that ci-framework users understand in case we need to override it?

Reply:
@fmount  `cifmw_cephadm_assimilate_conf_container ` is the path [1] to assimilate conf in the container , and we bind mount it in  [2]
can you suggest on what it should be renamed to ?  

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_cephadm/defaults/main.yml#L22
[2] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_cephadm/tasks/ceph_cli.yml#L29

Reply:
no I think we can keep as it is, maybe we can add a comment to indicate that is referring to the path within the container, but we can always follow up if we need.